src/components/sidebar/routePathView/LinkListView.tsx

```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { inject, observer } from 'mobx-react';
+import { SidebarStore } from '~/stores/sidebarStore';
+import { IRoutePath, INode, IRoutePathLink } from '~/models';
+import routeBuilder  from '~/routing/routeBuilder';
+import subSites from '~/routing/subSites';
+// import navigator from '~/routing/navigator';
+import NodeType from '~/enums/nodeType';
+import ButtonType from '~/enums/buttonType';
+import { Dropdown, Button } from '../../controls';
+import * as s from './linkListView.scss';
+// import classnames from 'classnames';
```

Comment:
Remove these comments if they are not used

Reply:
removed